.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: php
+
+sudo: false
+
+cache:
+  directories:
+    - "$HOME/.composer/cache"
+
+env:
+  - COMPOSER_FLAGS="--prefer-stable"
+
+matrix:
+  include:
+    - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
+    - php: 5.4
+    - php: 5.5
```

Comment:
I would keep them included using the normal way to build the matrix when there is nothing special to set, as done in MonologBundle: https://github.com/symfony/monolog-bundle/blob/master/.travis.yml